client/planning-extension/src/planning-details-widget.tsx

```diff
@@ -29,35 +29,65 @@ export class PlanningDetailsWidget extends React.PureComponent<IArticleSideWidge
         const {assignment_id} = this.props.article;
 
         if (assignment_id != null) {
-            extensionBridge.planning.getItemPlanningInfo({assignment_id})
-                .then((planning) => {
-                    this.setState({
-                        planningId: planning._id,
-                        loading: false,
-                    });
-                })
-                .catch(() => {
-                    this.setState({loading: false});
-                });
+            this.fetchPlanningInfo(assignment_id);
+            this.subscribeToUpdates();
         }
     }
 
+    componentWillUnmount() {
+        this.unsubscribeFromUpdates();
+    }
+
+    fetchPlanningInfo = (assignment_id: string) => {
+        extensionBridge.planning.getItemPlanningInfo({assignment_id})
+            .then((planning) => {
+                this.setState({
+                    planningId: planning._id,
+                    loading: false,
+                });
+            })
+            .catch(() => {
+                this.setState({loading: false});
+            });
+    };
+
+    subscribeToUpdates = () => {
+        (superdesk as any)?.events?.on?.('planning:updated', this.onPlanningUpdated);
```

Comment:
You shouldn't be casting `(superdesk as any)`. `superdesk.events` doesn't exist in the API, not sure how would it work?